CHANGELOG.md

```diff
@@ -3,6 +3,40 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v2.0.0 - Sapphire Duck [29/08/2024]
+
+New entrypoint option to skip isoseq pre-processing.
+Update the pipeline to nf-core 2.14.1.
+Update modules.
+Close (PR25)[https://github.com/nf-core/isoseq/issues/25]
+Close (PR27)[https://github.com/nf-core/isoseq/issues/27]
+Close (PR10)[https://github.com/nf-core/isoseq/issues/10]
```

Comment:
Could you consider write these as "Fixed error in test profile [PR25](Link to PR)" to see directly what was fixed?

Reply:
Sure :)